.github/workflows/scripts/monitor-typescript.ts

```diff
@@ -111,6 +119,7 @@ const transaction = Sentry.startTransaction({
 
 transaction.setName('typescript.compile');
 
+// @ts-ignore setMeasurements does not exist?
```

Comment:
@AbhiPrasad we have a ts error here, any thoughts on why it happens? The method works and I can see that [setMeasurements](https://github.com/getsentry/sentry-javascript/blob/master/packages/tracing/src/transaction.ts) is a public method on the transaction.

Reply:
```suggestion
// @ ts-expect-error setMeasurements does not exist?
```

so if we fix it later we'll get an error

Reply:
`setMeasurements` is not defined on the transaction interface, only on the class because it’s hidden/experimental at the moment.

https://github.com/getsentry/sentry-javascript/blob/a789ccd4e27e67d8ef1ffbc4bd50c377a32a66e4/packages/types/src/transaction.ts#L40

vs.

https://github.com/getsentry/sentry-javascript/blob/a789ccd4e27e67d8ef1ffbc4bd50c377a32a66e4/packages/tracing/src/transaction.ts#L73

we’ll update the types once we have custom measurements properly supported in the SDK - but you’ll have to TS ignore for now

Reply:
should we add a "TODO" here, so we don't forget to remove the TS ignore once `setMeasurements` is defined in the interface?